### PR TITLE
Prevent Countdown bar from hiding info on QT page

### PR DESCRIPTION
**Pull request by tomlovesgithub**
image

### CODE REVIEW

**Comment by HalcyonJAC** [comment]
Looks good to me :)

**Review by HalcyonJAC — APPROVED** [review]
I tested it locally and it looks good to me :)

File: src/components/QualifyingTest/Countdown2.vue [hunk 1]
```diff
@@ -184,6 +169,20 @@ export default {
 
 <style lang="scss" scoped>
 
+  @mixin mobile-view {
```
Comment (warrensearle) [inline src/components/QualifyingTest/Countdown2.vue:172]: Is this mixin used anywhere else?

If not then given this is scoped we could simplify as:
```
#time-remaining:before {
  content: 'Time remaining: ';
  @media (max-width: 599px) {
    content: '';
  }
}
```

**Review by warrensearle — APPROVED** [review]
Code looks alright to me 😄 
Will need some thorough browser testing

**Pull request merged**